src/mock_db/db.json

```diff
@@ -44,38 +44,50 @@
   "answer": {},
   "results": [
     {
-      "id": "f9674ec9-561d-269b-9a4d-94837ff81acc",
-      "value": "first result",
-      "answers": [
+    "title": "Pytanie single",
+    "data": {
+      "labels": ["January", "February", "March"],
+      "datasets": [
         {
-          "id": "f9674ec4-561d-269b-9a4d-94837ff81acc",
-          "value": "answer A",
-          "count": 3
-        },
-        {
-          "id": "f9674ec4-531d-269b-9a4d-94837ff81acc",
-          "value": "answer B",
-          "count": 4
+          "backgroundColor": "rgba(75,192,192,1)",
```

Comment:
Jak będziesz miał to juz w kodzie JSowym to daj tutaj kolor `theme.colors.primary[400]` bez żadnego bordera. Bardziej pasuje do reszty apki
![image](https://user-images.githubusercontent.com/39658211/81989085-e0af2400-963c-11ea-8682-4dcf5519e581.png)